x-pack/plugins/ml/public/application/routing/routes/new_job/index_or_search.tsx

```diff
@@ -72,6 +72,17 @@ const getLogCategorizationBreadcrumbs = (navigateToPath: NavigateToPath, basePat
   },
 ];
 
+const getChangePointDetectionBreadcrumbs = (navigateToPath: NavigateToPath, basePath: string) => [
+  getBreadcrumbWithUrlForApp('ML_BREADCRUMB', navigateToPath, basePath),
+  getBreadcrumbWithUrlForApp('AIOPS_BREADCRUMB_CHANGE_POINT_DETECTION', navigateToPath, basePath),
+  getBreadcrumbWithUrlForApp('CHANGE_POINT_DETECTION', navigateToPath, basePath),
```

Comment:
If I click on the `AIOps Labs` breadcrumb here, as a way to select a new data view, the route switches to the Explain Log Rate Spikes page.

<img width="582" alt="image" src="https://user-images.githubusercontent.com/7405507/199224316-3a3a2b69-10f3-4c16-a914-9e3c49f20f3f.png">

Reply:
fixed in https://github.com/elastic/kibana/pull/144093/commits/dbd1c5ef3f859f93c81b508fcb9bb57d88048352